### PR TITLE
feat(cli): build driver and upload to cdn

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -231,6 +231,7 @@ jobs:
       with:
         name: video-${{ matrix.browser }}-linux-test-results
         path: test-results
+
```
Comment (JoelEinbinder) [inline .github/workflows/tests.yml:234]:
```suggestion
```

File: utils/build-driver/upload-driver-32.ps1 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+$version = & "node -e ""console.log(require('../../package.json').version)"""
```
Comment (JoelEinbinder) [inline utils/build-driver/upload-driver-32.ps1:1]:
```suggestion
$version = & "node -p ""require('../../package.json').version"""
```

File: utils/build-driver/build-driver-64.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+set +x
+
+trap "cd $(pwd -P)" EXIT
+cd "$(dirname "$0")"
+
+PACKAGE_VERSION=$(node -e "console.log(require('../../package.json').version)")
```
Comment (JoelEinbinder) [inline utils/build-driver/build-driver-64.sh:8]:
```suggestion
PACKAGE_VERSION=$(node -p "require('../../package.json').version")
```

File: package.json [hunk 4]
```diff
@@ -31,7 +31,11 @@
     "roll-browser": "node utils/roll_browser.js",
     "coverage": "node test/checkCoverage.js",
     "check-deps": "node utils/check_deps.js",
-    "build-android-driver": "./utils/build_android_driver.sh"
+    "build-android-driver": "./utils/build_android_driver.sh",
+    "build-driver-64": "./utils/build-driver/build-driver-64.sh",
+    "build-driver-32": "@powershell -NoProfile -ExecutionPolicy Unrestricted -Command ./utils/build-driver/build-driver-32.ps1",
```
Comment (JoelEinbinder) [inline package.json:36]: Can we label this win32? It took me a bit to figure out that -64 runs on linux and -32 runs on windows.
```suggestion
    "build-driver-win32": "@powershell -NoProfile -ExecutionPolicy Unrestricted -Command ./utils/build-driver/build-driver-32.ps1",
```

File: .github/workflows/publish_canary_cdn.yml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+        registry-url: 'https://registry.npmjs.org'
+    - uses: microsoft/playwright-github-action@v1
+    - run: npm ci
+    - run: npm run build
+    - run: npm run build-driver-32
+    # - run: npm run upload-driver-32
```
Comment (JoelEinbinder) [inline .github/workflows/publish_canary_cdn.yml:44]: This is commented out because this is a draft?

File: .github/workflows/infra.yml [hunk 6]
```diff
@@ -23,3 +23,31 @@ jobs:
...
     - run: npm run lint
+
+  build-driver-all64:
+    name: "build-driver-all64"
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 10.15
```
Comment (mxschmitt) [inline .github/workflows/infra.yml:34]: nit: Its not fully related to this PR, but is there a reason why we are sometimes publishing with 10 and sometimes with 10.15?

Seems like currently only for Docker and linting 10.15 is used. So I would maybe use 10 which will result in the latest minor version.

**Review by aslushnikov — APPROVED** [review]

File: .github/workflows/infra.yml (hunk 6 above)
Comment (dgozman) [reply .github/workflows/infra.yml]: Done.

**Review by mxschmitt — APPROVED** [review]

**Pull request merged**